deny.toml

```diff
@@ -27,13 +27,17 @@ multiple-versions = "deny"
 wildcards = "allow"
 deny = []
 
-# Skip some multiple-versions checks, until they can be fixed.
-skip = [
-    { name = "ansi_term" }, # transitive dependencies only
-    { name = "env_logger" }, # pretty_env_logger and file-per-thread-logger depend on 0.7
-    { name = "humantime" }, # caused by env_logger
-    { name = "wast" }, # old one pulled in by witx
-    { name = "quick-error" }, # transitive dependencies
-    { name = "textwrap" }, # `miette` and `clap` depend on different versions
-    { name = "itoa" }, # `rustix` and `criterion` depend on different versions
+skip-tree = [
+    # Criterion 0.3 is pretty old at this point and has had an upcoming 0.4 for
+    # a long time. This is a dev-dependency so we don't really mind its
+    # dependency tree, so skip it entirely.
+    { name = "criterion", depth = 20 },
+
+    # This is maintained externally and we allow it to have duplicate
+    # dependencies relative to Wasmtime's main dependency tree.
+    { name = "witx", depth = 20 },
+
+    # This is somewhat unmaintained at this point and seems to pull in an old
+    # version of `env_logger`, so ignore it.
```

Comment:
@alexcrichton, are you sure this is correct? I still see activity in the repository and no slowdown in the number of downloads. It could be that it is just a stable project?

Reply:
I mostly say this in that the "upgrade env_logger" PR has been open for almost a year at this point

Reply:
Yeah, I saw that... But they're also still iterating on it (slowly). My comment was meant not to change this PR but as a "should I be using that crate?" question. I probably still will unless you think there's a better option out there.

Reply:
Oh I think it's fine to keep using, although personally I use `env_logger` where necessary (but choice-of-logger is a pretty personal thing at this point still I think, I don't think there's an obvious answer)